events/checks.go

```diff
@@ -0,0 +1,8 @@
+package events
+
+func NewCheckConsumerSync() SyncEventConsumer {
+	return SyncEventConsumer{
```

Comment:
@moshloop I initially tried this in an `init()` func but the upstream consumer
- needs a custom config
- and also might not have to be initiated at all.

Due to that I had to stick with this constructor pattern. However, the uniqueness is enforced now.